lib/music_album_interface.rb

```diff
@@ -0,0 +1,98 @@
+require 'json'
+require_relative 'music_album'
+require_relative 'genre_interface'
+
+class AlbumInterface
+  def initialize
+    @albumn_list = []
+    @genre_interface = GenreInterface.new
+  end
+
+  def create_album
+    print 'Enter a date [DD-MM-YYYY]: '
```

Comment:
- [Optional] I seems the format of date is different in different interfaces. It think our app will be more user friendly if we stick to one format. At other interfaces the date is in the format  [YYYY-MM-DD]